genkit-tools/cli/context/GENKIT.md

```diff
@@ -0,0 +1,394 @@
+# Genkit Node.js API Rules (v1.1x.x)
+
+This document provides rules and examples for building with the Genkit API in Node.js, using the `@genkit-ai/googleai` plugin.
+
+## Important Guidelines:
+
+- ALWAYS refer to documentation when available. Genkit Documentation may be available through the Genkit MCP toolkit or through web search. You may skip documentation check if you don't have access to these tools.
+
+- ONLY follow the specified project structure if starting a new project. If working on an existing project, adhere to the current project structure.
+
+## Core Setup
+
+1.  **Initialize Project**
+
+    ```bash
+    mkdir my-genkit-app && cd my-genkit-app
+    npm init -y
+    npm install -D typescript tsx @types/node
+    ```
+
+2.  **Install Dependencies**
+
+    ```bash
+    npm install genkit @genkit-ai/googleai data-urls node-fetch
+    ```
+
+3.  **Install Genkit CLI**
+
+    ```bash
+    npm install -g genkit-cli
+    ```
+
+4.  **Configure Genkit**
+
+    All code should be in a single `src/index.ts` file.
+
+    ```ts
+    // src/index.ts
+    import { genkit, z } from 'genkit';
+    import { googleAI } from '@genkit-ai/googleai';
+
+    export const ai = genkit({
+      plugins: [googleAI()],
+    });
+    ```
```

Comment:
Aside from the package names, do we need to go into dfetails on how to install and init a project with genkit in this file?